### ISSUE TITLE
:qa:wildfly:startWildfly Failed to find http port in log

### ISSUE THREAD

**Issue by matriv** [elastic/elasticsearch#53574]
Failed in 7.x:

```
19:58:18 Caused by: org.gradle.api.GradleException: Failed to find http port in wildfly log
19:58:18 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
19:58:18 	at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
19:58:18 	at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
19:58:18 	at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:500)
19:58:18 	at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:481)
19:58:18 	at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:80)
19:58:18 	at org.codehaus.groovy.reflection.CachedConstructor.doConstructorInvoke(CachedConstructor.java:74)
19:58:18 	at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrap.callConstructor(ConstructorSite.java:84)
19:58:18 	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:59)
19:58:18 	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:237)
19:58:18 	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:249)
19:58:18 	at build_d2vtzzgui3wbhjey5k6ql5wkn$_run_closure8$_closure26$_closure27.doCall(/dev/shm/elastic+elasticsearch+7.x+multijob+fast+part1/qa/wildfly/build.gradle:169)
19:58:18 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
19:58:18 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
19:58:18 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
19:58:18 	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
19:58:18 	at org.codehaus.groovy.reflection.CachedMethod.invoke(CachedMethod.java:101)
19:58:18 	at groovy.lang.MetaMethod.doMethodInvoke(MetaMethod.java:323)
19:58:18 	at org.codehaus.groovy.runtime.metaclass.ClosureMetaClass.invokeMethod(ClosureMetaClass.java:263)
19:58:18 	at groovy.lang.MetaClassImpl.invokeMethod(MetaClassImpl.java:1041)
19:58:18 	at groovy.lang.Closure.call(Closure.java:405)
19:58:18 	at groovy.lang.Closure.call(Closure.java:421)
19:58:18 	at org.codehaus.groovy.runtime.IOGroovyMethods.withReader(IOGroovyMethods.java:1161)
```

Build scan: https://gradle-enterprise.elastic.co/s/qncqcqbmeqgly

Cannot reproduce locally.

There was a change with #53126 to use docker.

**Comment by jasontedor** [comment]
This will be closed by the backport PR: #53159.

**Comment by rjernst** [comment]
closed by #53159
